tasks/util/constants.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     pathToJasmineTests: path.join(pathToRoot, 'test/jasmine/tests'),
     pathToJasmineBundleTests: path.join(pathToRoot, 'test/jasmine/bundle_tests'),
 
+    mapboxAccessToken: 'pk.eyJ1IjoiZXRwaW5hcmQiLCJhIjoiY2luMHIzdHE0MGFxNXVubTRxczZ2YmUxaCJ9.hwWZful0U2CQxit4ItNsiQ',
```

Comment:
Maybe we should add a comment that _we're aware there's a token here and that we're ok with it_ so we don't have too many overzealous contributions to improve security?